### ISSUE TITLE
-j and --json flag is missing in 1.2.5

### ISSUE THREAD

**Issue by nitrocode** [sibprogrammer/xq#112]
**Problem Statement**

-j and --json flag is missing in 1.2.5

**Steps to Reproduce**

```
✗ brew install xq
✗ xq --version
xq version 1.2.5
✗ xq --help
Command-line XML and HTML beautifier and content extractor

Usage:
  xq [flags]

Flags:
  -a, --attr string      Extract an attribute value instead of node content for provided CSS query
  -c, --color            Force colorful output
  -e, --extract string   Extract a single node from XML
  -h, --help             Print this help message
  -m, --html             Use HTML formatter
      --indent int       Use the given number of spaces for indentation (default 2)
      --no-color         Disable colorful output
  -n, --node             Return the node content instead of text
  -q, --query string     Extract the node(s) using CSS selector
      --tab              Use tabs for indentation
  -v, --version          Print version information
  -x, --xpath string     Extract the node(s) from XML
```

**Actual Result**

```
✗ cat ~/Downloads/test.xml| xq -j
Error: unknown shorthand flag: 'j' in -j
```

**Expected Result**

Outputted JSON

**Comment by sibprogrammer** [comment]
This is not a bug. This option is available only in 1.3.0, which is still in development (but I think it will be released soon). README can be a little bit misleading because of the absence of versioning.

**Comment by lamyergeier** [comment]
> This is not a bug. This option is available only in 1.3.0, which is still in development (but I think it will be released soon). README can be a little bit misleading because of the absence of versioning.

@sibprogrammer: Is 1.3.0 the next version or 5 releases away? When can we expect it tentatively (in 3 months, 6 months, 1 year)?

**Comment by sibprogrammer** [comment]
v1.3.0 is out :)
